Python/ToonifyImage/toonify.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import shutil # to save image locally
+imagePath=input("Please enter the path of image to toonify : ")
+api_key=input("Please enter your api key")

```

Comment:
Please have the API Key in a `.env` file and use a module like dotenv to import them in your Python Code